docs/index.rst

```diff
@@ -3411,18 +3411,69 @@ The only required argument to create an ensemble is the dataset id:
 
     ensemble = api.create_ensemble('dataset/5143a51a37203f2cf7000972')
 
-but you can also specify the number of models to be built and the
-parallelism level for the task:
+BigML offers three kinds of ensembles. Two of them are known as ``Decision
+Forests`` because they are built as collections of ``Decision trees``
+whose predictions
+are aggregated using different combiners: ``plurality``,
```

Comment:
this colon looks misplaced... seems the combiners list should be between parens, no? "diferent combiners (plurality, ...) to issue the ensemble's prediction."